scripts/deploy.sh

```diff
@@ -63,12 +63,27 @@ if [ "${DEPLOY_AIO}" != false ]; then
     # RO-3316 has been resolved.
     openstack-ansible -i 'localhost,' \
                       -e 'apt_target_group=localhost' \
+                      -e 'apt_artifact_mode="${RPCO_APT_ARTIFACTS_MODE}"' \
                       -e 'container_artifact_enabled=false' \
                       "${SCRIPT_PATH}/../playbooks/site-artifacts.yml"
 
     # Install OpenStack-Ansible
     openstack-ansible "${SCRIPT_PATH}/../playbooks/openstack-ansible-install.yml"
 
+    # Set the AIO config bootstrap options
+    apt_artifact_enabled=$(awk '/apt_artifact_enabled/ {print $3}' /etc/ansible/facts.d/rpc_openstack.fact)
+    if [[ "${apt_artifact_enabled}" == "True" ]]; then
+      if [[ "${RPCO_APT_ARTIFACTS_MODE}" == "strict" ]]; then
+        # Prevent the AIO bootstrap from re-implementing
+        # the updates, backports and UCA sources.
+        export BOOTSTRAP_OPTS='{ "bootstrap_host_apt_distribution_suffix_list": [], "uca_enable": "False" }'
```

Comment:
Understandably a mix of json and string options isn't working, and the gate test uses a flavor with a large secondary disk. This will likely require an upstream patch - working on it now.